webapp/channels/src/i18n/en_AU.json

```diff
@@ -5044,14 +5044,14 @@
   "status_dropdown.profile_button_label.online": "Current status: Online. Select to open Profile and Status menu.",
   "status_dropdown.profile_button_label.ooo": "Current status: Out of office. Select to open Profile and Status menu.",
   "status_dropdown.set_away": "Away",
-  "status_dropdown.set_custom": "Set a Custom Status",
-  "status_dropdown.set_custom_text": "Set Custom Status Text",
-  "status_dropdown.set_dnd": "Do Not Disturb",
+  "status_dropdown.set_custom": "Set a custom status",
```

Comment:
Could you actually revert the changes to this file? Our translation system tends to run into conflicts if files other than `en.json` are updated directly in the repo

Reply:
Done 👍🏻